utils/ios.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -e
+
+if ! [ -f LICENSE ] || ! [ -d include/session ]; then
+    echo "You need to run this as ./utils/ios.sh from the top-level libsession-util project directory"
+    exit 1
+fi
+
+if ! command -v xcodebuild; then
+    echo "xcodebuild not found; are you on macOS with Xcode and Xcode command-line tools installed?" >&2
+    exit 1
+fi
+
+archive="${1:-libsession-util-ios-TAG.tar.xz}"
+
+if [[ "$archive" =~ TAG ]]; then
+    if [ -n "$DRONE_TAG" ]; then
+        tag="$DRONE_TAG"
+    elif [ -n "$DRONE_COMMIT" ]; then
+        tag="$(date --date=@$DRONE_BUILD_CREATED +%Y%m%dT%H%M%SZ)-${DRONE_COMMIT:0:9}"
+    else
+        tag="$(date +%Y%m%dT%H%M%SZ)-$(git rev-parse --short=9 HEAD)"
+    fi
+
+    archive="${archive/TAG/$tag}"
+fi
+
+
+set -x
+
+for i in "ios-arm64 OS64" "sim-arm64 SIMULATORARM64" "sim-x86_64 SIMULATOR64"; do
+    i=($i)
+    build="build-ios/${i[0]}"
+    platform="${i[1]}"
+    echo "Building ${i[0]} for $platform in $build"
+
+    ./utils/static-bundle.sh "$build" "" \
+        -DCMAKE_TOOLCHAIN_FILE=../../external/ios-cmake/ios.toolchain.cmake \
```

Comment:
use absolute path here as having it be a relative the way it is hard to understand.

Reply:
Relative is easier when you are running the same commands manually while looking at the script (e.g. to debug something).